framework/db/sqlite/Schema.php

```
@@ -84,23 +92,81 @@ class Schema extends BaseSchema implements ConstraintFinderInterface
+     * @since 2.0.56
+     */
+    protected function resolveTableNames($table, $name)
+    {
+        $parts = explode('.', str_replace(['`', '"'], '', $name));
+        if (isset($parts[1])) {
+            $table->schemaName = $parts[0];
+            $table->name = $parts[1];
+        } else {
+            $table->schemaName = $this->defaultSchema;
+            $table->name = $parts[0];
+        }
```

PHPMD (2.15.0)

[error] The method resolveTableNames uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)